### PR TITLE
fix not show every nutrient but only 3 principals

### PR DESCRIPTION
**Pull request by antomax33**
postinfo_title

### CODE REVIEW

File: app/src/main/java/com/github/se/polyfit/ui/components/postinfo/Text.kt [hunk 1]
```diff
@@ -22,25 +22,29 @@ import com.github.se.polyfit.model.post.Post
...
 
   TextPost(
       text = ContextCompat.getString(LocalContext.current, R.string.nutrient),
       modifier = modifier.testTag("NutrientTitle"),
       bold = true)
 
-  post.meal.ingredients.forEach { it.nutritionalInformation.nutrients.forEach { NutrientInfo(it) } }
+  post.getCarbs()?.let { NutrientInfo(it) }
+  post.getFat()?.let { NutrientInfo(it) }
+  post.getProtein()?.let { NutrientInfo(it) }
```
Comment (medwardson) [inline app/src/main/java/com/github/se/polyfit/ui/components/postinfo/Text.kt:47]: perhaps you can use
`post.meal.getMacros()` instead?

**Review by medwardson — APPROVED** [review]

File: app/src/main/java/com/github/se/polyfit/ui/components/postinfo/Text.kt (hunk 1 above)
Comment (antomax33) [reply app/src/main/java/com/github/se/polyfit/ui/components/postinfo/Text.kt:47]: Didn't knew that.

**Review by Neb1010 — COMMENTED** [review]
Good implementation to have only the most important Meal's infos, and to no have a post 2miles long

**Pull request merged**